cabinets/cabinet/__init__.py

```diff
@@ -38,18 +39,34 @@ def set_configuration(cls, **kwargs):
         pass  # pragma: no cover
 
     @classmethod
-    def read(cls, path, raw=False, **kwargs):
-        if raw:
-            return cls.read_content(path)
-        else:
-            return Parser.load(path, cls.read_content(path), **kwargs)
+    def read(cls, path, parser: Union[bool, Type[Parser]] = True, **kwargs):
+        if parser is True:
+            return Parser.load(path, cls.read_content(path, **kwargs), **kwargs)
+        if parser is False:
+            return cls.read_content(path, **kwargs)
+        try:
+            if issubclass(parser, Parser):
+                return parser.load_content(cls.read_content(path, **kwargs), **kwargs)
+        except TypeError:
+            pass
```

Comment:
I feel like this organizes things a bit better. The value is only ever going to match one of these cases, so I think `elif`s are more appropriate. I also don't think the subclass check should live in the `try`/`except` block. 
```suggestion
        if parser is True:
            return Parser.load(path, cls.read_content(path, **kwargs), **kwargs)
        elif parser is False:
            return cls.read_content(path, **kwargs)
        elif issubclass(parser, Parser):
            try:
                return parser.load_content(cls.read_content(path, **kwargs), **kwargs)
            except TypeError:
                pass
```

Reply:
Actually the whole purpose of the `try/except` is to catch `TypeErrors` from `issubclass` -- `issubclass` will throw a `TypeError` if arg 1 is not a class (is an object). I'm fine with the organization `elif` change though.

Reply:
Not being able to move the `issubclass` check up to an `elif` statement makes this reorganization less nice...

Reply:
Interesting...and if `load_content()` fails you want it crash? Also how would you feel about just using `inspect.isclass(parser)` as part of the third condition instead of try/except. I personally prefer the parity of the three clearly defined cases.

Reply:
Yes I think we should allow errors to propagate. I am down with the `inspect` check, since it allows us to let `TypeError`s propagate too.